code/modules/client/preference_setup/loadout/items/factions.dm

```diff
@@ -356,7 +356,7 @@
 	pmcg_headwear["EPMC softcap"] = /obj/item/clothing/head/softcap/pmc/epmc
 	pmcg_headwear["PMCG woolen hat"] = /obj/item/clothing/head/wool/pmc
 	pmcg_headwear["EPMC woolen hat"] = /obj/item/clothing/head/wool/pmc/alt
-	pmcg_headwear["PMCG garrison cap"] = /obj/item/clothing/head/sidecap/pmcg
+	pmcg_headwear["PMCG side cap"] = /obj/item/clothing/head/sidecap/pmcg
```

Comment:
inconsistent grammar: you've written "sidecap" in other places

Reply:
i'm using side cap with a space as the standard now